Client/src/Components/Graph/Graph.js

```diff
@@ -128,7 +108,40 @@ class Graph extends Component {
             .attr("cy", function (d) {
                 return yScale(d["y"]);
             })
+            .attr("clip-path", "")
```

Comment:
You can remove this part too, i dont think it needs to be here the clip path attr